src/main/kotlin/io/github/nomisrev/routes/articles.kt

```diff
@@ -41,6 +71,35 @@ data class Comment(
   val author: Profile
 )
 
+@Resource("/article")
+data class ArticleResource(val parent: RootResource = RootResource) {
+  @Resource("/feed") data class Feed(val parent: ArticleResource = ArticleResource())
```

Comment:
The feed has 2 query parameters, and we should encode them into the [type-safe route](https://ktor.io/docs/type-safe-routing.html#resource_query_param).

```
 - name: offsetParam
         in: query
         name: offset
         required: false
         schema:
           type: integer
           minimum: 0
         description: The number of items to skip before starting to collect the result set.
       - name: limitParam
         in: query
         name: limit
         required: false
         schema:
           type: integer
           minimum: 1
           default: 20
         description: The numbers of items to return.
```

```suggestion
  @Resource("/feed")
  data class Feed(
    val offsetParam: Int,
    val limitParam: Int = 20,
    val parent: ArticleResource = ArticleResource()
  )
```